Update MinLength description (#2058)
telerik/blazor-docs: components/multiselect/overview.md

| `FilterOperator` | `StringFilterOperator` <br /> (`StartsWith`) | The string operation that will be used for [filtering]({%slug multiselect-filter%}). |
| `Id` | `string` | Renders as the `id` attribute on the `<select />` element, so you can attach a `<label for="">` to it. |
| `LoaderShowDelay` | `int ` <br /> 300 | Time in milliseconds between opening the popup and showing the loading skeleton in it when the data is not yet available. |

> | `MinLength` | `int` | How many characters the text has to be before the suggestions list appears. Often works together with [filtering]({%slug multiselect-filter%}). |

Reviewer: When a noun is used as an adjective, it is used in singular form.

```suggestion
| `MinLength` | `int` | How many characters the user must type before the suggestion list appears. Often works together with [filtering]({%slug multiselect-filter%}). |
```
